app/models/event_user.rb

```diff
@@ -1,6 +1,6 @@
 class EventUser < ActiveRecord::Base
   # TODO Do we need these roles?
-  ROLES = [['Speaker', 'speaker'], ['Submitter', 'submitter'], ['Moderator', 'moderator']]
+  ROLES = [%w[Speaker speaker], %w[Submitter submitter], %w[Moderator moderator]]
```

Comment:
@differentreality `%w()` is the default for this cop. Or I don't understand what you mean. :thinking: